squeasel.c

```diff
@@ -3595,7 +3595,8 @@ static int is_valid_port(unsigned int port) {
 // Examples: 80, 443s, 127.0.0.1:3128, 1.2.3.4:8080s
```

Comment:
Perhaps update the `[ip_address:]port[s]` to `[ip_address:]port[s][r]` above and explain what `s` and `r` mean?